test/pipeline/pipeline.import.azure.master.json

```diff
@@ -210,18 +516,19 @@
                             "targetType": "inline",
                             "filePath": "",
                             "arguments": "",
-                            "script": "New-Item '$(ttk.folder)' -ItemType Directory\nInvoke-WebRequest -uri '$(ttk.uri)' -OutFile \"$(ttk.folder)/$(ttk.asset.filename)\" -Verbose\nGet-ChildItem '$(ttk.folder)' -Recurse\n\nWrite-Host \"Expanding files...\"\nExpand-Archive -Path '$(ttk.folder)/*.zip' -DestinationPath '$(ttk.folder)' -Verbose\n\nWrite-Host \"Expanded files found:\"\nGet-ChildItem '$(ttk.folder)' -Recurse",
+                            "script": "$ttkUri = \"$(ttk.uri)\"\nwrite-host $ENV:BUILD_REPOSITORY_ID\nwrite-host $ENV:SYSTEM_PULLREQUEST_PULLREQUESTNUMBER\n\n# Look for use-staging-scripts label\n$labelUri = \"https://api.github.com/repos/$ENV:BUILD_REPOSITORY_ID/issues/$ENV:SYSTEM_PULLREQUEST_PULLREQUESTNUMBER/labels\"\nWrite-Host \"Looking for label at $labelUri\"\n$labelResult = curl -s $labelUri\nWrite-Host $labelResult\nif ($labelResult -like '*\"name\": \"use-staging-scripts\"*') {\n    Write-Host \"... Found\"\n    $ttkUri = \"https://azurequickstartsservice.blob.core.windows.net/ttk/staging/arm-template-toolkit.zip\"\n    Write-Host \"Using script location $ttkUri\"\n} else {\n    Write-Host \"... not found\"\n}\n\nNew-Item '$(ttk.folder)' -ItemType Directory\nWrite-Host \"ttkUri: $ttkUri\"\nInvoke-WebRequest -uri $ttkUri -OutFile \"$(ttk.folder)/$(ttk.asset.filename)\" -Verbose\nGet-ChildItem '$(ttk.folder)' -Recurse\n\nWrite-Host \"Expanding files...\"\nExpand-Archive -Path '$(ttk.folder)/*.zip' -DestinationPath '$(ttk.folder)' -Verbose\n\nWrite-Host \"Expanded files found:\"\nGet-ChildItem '$(ttk.folder)' -Recurse",
```

Comment:
Inserted this at top of script:
```ps
$ttkUri = "$(ttk.uri)"
write-host $ENV:BUILD_REPOSITORY_ID
write-host $ENV:SYSTEM_PULLREQUEST_PULLREQUESTNUMBER

# Look for use-staging-scripts label
$labelUri = "https://api.github.com/repos/$ENV:BUILD_REPOSITORY_ID/issues/$ENV:SYSTEM_PULLREQUEST_PULLREQUESTNUMBER/labels"
Write-Host "Looking for label at $labelUri"
$labelResult = curl -s $labelUri
Write-Host $labelResult
if ($labelResult -like '*"name": "use-staging-scripts"*') {
    Write-Host "... Found"
    $ttkUri = "https://azurequickstartsservice.blob.core.windows.net/ttk/staging/arm-template-toolkit.zip"
    Write-Host "Using script location $ttkUri"
} else {
    Write-Host "... not found"
}
```
Just add the "use-staging-scripts" label to a PR and it will use the current staging script instead of published.

Reply:
Could add a ttk.staging.uri variable for https://azurequickstartsservice.blob.core.windows.net/ttk/staging/arm-template-toolkit.zip if prefer

Reply:
seems like we should add a var for the uri